notflix

```diff
@@ -9,7 +9,8 @@ baseurl="https://1337x.wtf"
 cachedir="$HOME/.cache/notflix"
 
 if [ -z $1 ]; then
-  query=$(dmenu -p "Search Torrent: " <&-)
+  echo "Search for Shows:  "
+  read query
```

Comment:
I want to keep using dmenu. You can set a verification if notflix is started from terminal to ask for CLI input or if it is started from background (like when using dmenu_run) to use dmenu:
```
	if [ $(ps -o stat= -p $$) = "S+" ]; then
		echo "Search for Shows: "
		read query
	else
		query=$(dmenu -p "Search Torrent: " <&-)
	fi
```
That is the best for me but you can simply check if user is having dmenu installed like:
```
[ -x "$(command -v dmenu)" ]
```